<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participant</class>
 <widget class="QDialog" name="Participant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 103, 136);</string>
  </property>
  <widget class="QPushButton" name="clear_lines">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color:  rgb(72, 28, 28);
border: none;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_participantList">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 32, 57);
border: none;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_participant_data">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 32, 57);
border: none;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insert_participant">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 85, 45);
border: none;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>201</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(17, 20, 63);
border-bottom-color: rgb(17, 20, 63);
border-right-color: rgb(17, 20, 63);
gridline-color: rgb(17, 20, 63);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Mentors</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="listView_mentors">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(229, 229, 229);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Mentees</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="listView_mentees">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(229, 229, 229);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="addParticipant_form">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Major</string>
    </property>
   </widget>
   <widget class="QComboBox" name="choose_major">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computer Information Systems</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computer Network Technology</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computer Science</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Engeneering Science</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Geographyc Information Science</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Topic</string>
    </property>
   </widget>
   <widget class="QComboBox" name="topic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(223, 223, 223);</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clean Water</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Good Health </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green Consumption</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quality Education</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Racial Justice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zero Hunger</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="group_id">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mentor_role">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mentor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mentee_role">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mentee</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Company</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="company">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="delete_selected">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color:  rgb(72, 28, 28);
border: none;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="participantBack_button">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>590</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(117, 117, 117);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
